admin.php

```diff
@@ -41,218 +41,191 @@ function sqlQuery($statement, $link)
 ?>
 <html>
 <head>
-<title>OpenEMR Site Administration</title>
-<link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css" type="text/css">
-<script type="text/javascript" src="public/assets/jquery/dist/jquery.min.js"></script>
-<script type="text/javascript" src="public/assets/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-<link rel="stylesheet" href="public/assets/@fortawesome/fontawesome-free/css/all.min.css" type="text/css">
-<link rel="shortcut icon" href="public/images/favicon.ico" />
-<style>
-    .oe-pull-away{
-        float:right;
-    }
-    .oe-help-x {
-        color: grey;
-        padding: 0 5px;
-    }
-    .oe-superscript {
-        position: relative;
-        top: -.5em;
-        font-size: 70%!important;
-    }
-    .oe-setup-legend{
-        background-color:  WHITESMOKE;
-        padding:0 10px;
-    }
-    .oe-text-green {
-        color: green;
-    }
-    button {
-    font-weight:bold;
-    }
-    .button-wait {
-        color: grey;
-        cursor: not-allowed;
-        opacity: 0.6;
-    }
-    @media only screen {
-        fieldset > [class*="col-"] {
-            width: 100%;
-            text-align:left!Important;
-        }
-    }
-</style>
+    <title>OpenEMR Site Administration</title>
+    <link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css">
+    <script src="public/assets/jquery/dist/jquery.min.js"></script>
+    <script src="public/assets/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="public/assets/@fortawesome/fontawesome-free/css/all.min.css">
+    <link rel="shortcut icon" href="public/images/favicon.ico" />
 </head>
 <body>
-    <div class='container'>
+    <div class='container mt-3'>
         <div class="row">
-            <div class="col-sm-12">
-                <div class="page-header">
-                    <h2>OpenEMR Multi Site Administration <a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="Click to view Help"><i class="fa fa-question-circle" aria-hidden="true"></i></a></h2>
+            <div class="col-12">
+                <div class="page-header d-flex justify-content-between align-items-center">
+                    <h2>OpenEMR Multi Site Administration</h2>
```

Comment:
Hi @tywrenn 

Looks like the `.page-header` the class was deprecated in Bootstrap 4.0
Should I remove all `.page-header` class?

https://getbootstrap.com/docs/4.0/migration/#typography

Reply:
hi @stu01509 , Willing to bet @tywrenn would want you to remove the `.page-header` class on the scripts as you work on them (rather than trying to change them all at once).

Reply:
So would remove it here and then use the utility classes instead to make it look nice.

Reply:
Updated the commit, please review it again.
Also, should I create a PR to remove the `.page-header` class?